bank.cpp

```diff
@@ -0,0 +1,180 @@
+#include "bank.h"
+
+Bank::Bank()
+{
+}
+
+void Bank::create_account()
+{
+    json j;
+    Person ps;
+    out.open("bankfile.json", std::ios::app);
+
+    if (!out.is_open()) {
+        std::cout << "Cannot open file!\n";
+        exit(1);
+    }
+
+    std::cout << "Please enter your name \n";
+    std::cin >> ps.name;
+    std::cout << "Please enter your last name \n";
+    std::cin >> ps.last_name;
+    std::cout << "Please enter your middle name \n";
+    std::cin >> ps.middle_name;
+    std::cout << "Please enter your passport number \n";
+    std::cin >> ps.passport_number;
+    std::cin.ignore();
+    std::cout << "Please enter your date of birth \n";
+    getline(std::cin, ps.date_of_birth);
+    std::cout << "Please enter your deposit \n";
+    std::cin >> ps.money;
+
+    account_number++;
+
+    std::string staccount_number = ps.passport_number;
+
+    j[ps.passport_number] = {
+        { " Name", ps.name },
+        { " Middle Name", ps.middle_name },
+        { " Last Name", ps.last_name },
+        { " Date of birth", ps.date_of_birth },
+        { " Deposit", ps.money }
+    };
+
+    out << j.dump(4);
+
+    out.close();
+
+    std::cout << "Your account created successesfully !\n";
+}
+
+void Bank::update_inf_exs_account()
+{
+
+    std::fstream finout;
+    finout.open("bankfile.json", std::fstream::in | std::fstream::out);
+
+    std::ofstream temp;
+    temp.open("temp.json"); //delete file
```

Comment:
Delete 58 and 59 lines and change "temp.open("temp.json", std::ios::app)" to temp.open("temp.json", std::ios::out)